project/api/views.py

```diff
@@ -88,6 +88,27 @@ def get_all_request():
     return jsonify(response), 200
 
 
+@request_blueprint.route("/requests/available/<usermail>", methods=["GET"])
```

Comment:
Alterar a forma que foi feita aqui:
https://github.com/Lend-it/Request/blob/devel/project/api/views.py#L58

`/requests?usermail=lucas@email.com`